### PR TITLE
generalize deus ex

### PR DESCRIPTION
**Pull request by TerryHowe**

### CODE REVIEW

**Comment by bscofield** [comment]
Thanks for this! Will take a look over the weekend.

**Comment by TerryHowe** [comment]
Forgot about this, you too maybe?

**Comment by bscofield** [comment]
Haven't forgotten; have just been stupidly busy. Sorry for the delay!

File: README.md [hunk 1]
```diff
@@ -17,13 +17,27 @@ Or install it yourself as:
...
+:server_create:
+  :flavor_ref: '103'
+  :image_ref: '8c096c29-a666-4b82-99c4-c77dc70cfb40'
+  :key_name: 'bover'
+  :nics: [ 'net_id': '76abe0b1-581a-4698-b200-a2e890f4eb8d' ]
+:floating_ip_create:
+  :floating_network_id: '7da74520-9d5e-427b-a508-213c84e69616'
+:public_key_path: /home/terry/.ssh/bover.pub
+:private_key_path: /home/terry/.ssh/bover.pem
+:username: ubuntu
```
Comment (bscofield) [inline README.md]: First thought: this seems way more complicated to setup than the AWS-only version. My original goal was that this should be almost drop-in easy. Is there anything simpler we can do here?

Comment (TerryHowe) [inline README.md]: I hear you, but it is really mostly the same values shuffled into :authentication and :server_create.  If the provider automatically assigns a public IP, there would be no need for :floating_ip_create or :nics.  The :key_name is not required if you wanted to use the default keypair.

How about I change it to an AWS example and maybe have an OpenStack one after that.

Comment (bscofield) [inline README.md]: Sounds good. Have the first example be as unscary as possible :smile:

**Comment by TerryHowe** [comment]
Alright, README simplified and tested in AWS

**Comment by bscofield** [comment]
OK, thanks!

**Pull request merged**